contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue

```diff
@@ -500,12 +502,26 @@
       newContent() {
         return !this.nodes.some(n => n.isNew);
       },
+      debouncedUpdate() {
+        this.diffTracker;
```

Comment:
Doing this inside a computed property feels a bit off. Could we just put a `watch` on `diffTracker` instead to do the debounce?

Looking at it, I think it means a new debounced function will be created every time diffTracker gets updated, so a previous call will not be prevented, because each time it changes, this will get recomputed, and a new debounced function returned.

(will probably need to be a deep watch as diffTracker is nested here)

Reply:
I'll remove that line since it's definitely not needed (and might cause what you're describing). However, after I removed it, I added a print statement, and it looks like it's working as intended as a computed property. One complication with using a watch function is that it deletes the ids as it makes the updates, which seems like it would cause some recursion on the watcher